lib/material/material.flow

```diff
@@ -999,7 +999,7 @@ export {
 								MDataTableScrollStyle, TScrollPosition, TScrollEnabled;
 
 			MDataTableStyle ::= MLightweightDataTableStyle, MListReorder /*Disables pagination and sorting*/, MListReorderHideIcon, MReorderOnDragEnd,
-								MFixedRow, MExpandRow;
+								MFixedRow, MExpandRow, MTableExpander;
```

Comment:
it shouldn't be the part of MDataTableStyle, if it is not implemented for MDataTable in general